assets/js/app.js

```diff
@@ -2,7 +2,12 @@
  * Main entry file for app logic
  */
 
+// Config
+app.config = {

```

Comment:
Oke. Breakpoints kan, maar ik vergroot hem ook wel eens naar andere site-specifieke variabele die door de site gebruikt worden en dus ook in JS beschikbaar worden. Eens dat breakpoints toegevoegd kan worden. Zal er nog eens over nadenken.